### DISCUSSION TITLE
SENT BASE TO A WRONG ADDRESS

### DISCUSSION THREAD

**Discussion by Thisisdwayne** [blockscout/blockscout#8378] [Support] [answered]
IS THERE ANYTHING I CAN DO??

**Comment by ulyanas** [comment, marked as answer]
Blockchain transactions are not reversible. It is not possible to "undo" or "reverse" a transaction once it is confirmed by the network. You can try contacting Base's support channel on Discord for further clarifications https://discord.gg/buildonbase

**Comment by tripathirishabh47** [comment]
Yesterday I requested for withdrawal of USDT, it is showing success but the status has not reached my account.
